src/main/java/com/microsoft/jenkins/containeragents/aci/AciCloud.java

```diff
@@ -43,6 +49,8 @@
 
     private String credentialsId;
 
+    private String logAnalyticsCredentialsId;
```

Comment:
Thoughts on adding an integration test for this functionality?

Example here: https://github.com/jenkinsci/azure-container-agents-plugin/blob/master/src/test/java/com/microsoft/jenkins/containeragents/AciCloudIT.java